src/handler.ts

```diff
@@ -110,20 +116,49 @@ const processLoginAction = async (
         await _page.focus('#i0118');
         await _page.keyboard.type(_password);
         logger.info(`Logging with password ${'*'.repeat(_password.length)}`);
-        await _page.waitForSelector('input[value="Sign in"]', { timeout: 60000 });
-        await _page.waitForSelector('input[value="Sign in"]', { timeout: 60000 });
-        await _page.click('input[value="Sign in"]');
+
+        if (is_test) {
+            await _page.waitForTimeout(2000);
+            await _page.waitForSelector('input[value="登入"]', { timeout: 60000 });
+            await _page.click('input[value="登入"]');
+        }
+
+        else {
+            await _page.waitForSelector('input[value="Sign in"]', { timeout: 60000 });
+            await _page.click('input[value="Sign in"]');
+        }
+
```

Comment:
- `if..else` indent, should be like below
```
if (xxxxx) {
    ....
} else {
    ....
}
```

- in `testing` mode, language should be `english`

![image](https://user-images.githubusercontent.com/18332044/128839345-44c01ae2-a29a-44ee-9591-b505c9860e21.png)